### DISCUSSION TITLE
Batch convert OCR to SRT

### DISCUSSION THREAD

**Discussion by voska7** [SubtitleEdit/subtitleedit#6360] [Q&A]
there is multiple option for OCR to SRT convert. for batch convert how can I set which method subtitle edit will use? i could not find any option.

**Comment by niksedk** [comment]
You can right-click in the input file list view:

![image](https://user-images.githubusercontent.com/1261331/198047534-1b67d7a9-60d1-4592-aa42-4cd4970653c9.png)

**Comment by voska7** [comment]
default tesseract using 3.02 method. can we have other ocr method options please? i would like to use google api.

**Comment by voska7** [comment]
also lets say we want to use tesseract 5.2.0 and other engine modes+ music symbol etc option enable, how can we do that? it would be really nice if we can have those option in batch mode.

**Reply by niksedk** [reply]
SE uses the current settings for Tesseract/nOCR

**Reply by voska7** [reply]
thank you, but it seems google vision API settings is not working for batch conversion. also, google vision API cannot detect italics.  and thank you so much for the latest update.
